test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -270,9 +270,44 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
         $result | should be ([char]0x1b)
     }
 
-	It "Test that escaping any character with no special meaning just returns that char. (line 602)" {
-        $result = ExecuteCommand '"fo`obar"'
-		$result | should be "foobar"
+    Context "Test Unicode escape sequences." {
+        # These tests require the file to be saved with a BOM.  Unfortunately when this UTF8 file is read by
+        # PowerShell without a BOM, the file is incorrectly interpreted as ASCII.
+        It 'Test that Unicode escape sequence `u2195 returns the ↕ character.' {
+            $result = ExecuteCommand '"foo`u2195abc"'

```

Comment:
Good catch! Thanks.  Was debugging utf8 encoding problems with the file and removed this one by accident.